src/analysis/retail/druid/balance/modules/spells/Starweaver.tsx

```diff
@@ -0,0 +1,90 @@
+import { formatNumber } from 'common/format';
+import SPELLS from 'common/SPELLS';
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events, { CastEvent } from 'parser/core/Events';
+import BoringSpellValueText from 'parser/ui/BoringSpellValueText';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
+import { TALENTS_DRUID } from 'common/TALENTS';
+import { SpellLink } from 'interface';
+
+const STARWEAVER = {
+  STARSURGE_COST: 40,
```

Comment:
Pull these from the druid constants file

Reply:
I'd also prefer this came from `balance/constants.ts`, but otherwise am fine with merging this.